single.php

```diff
@@ -1,20 +1,23 @@
 <?php
+
 /**
  * The Template for displaying all single posts
- *
- * Methods for TimberHelper can be found in the /lib sub-directory
- *
- * @package  WordPress
- * @subpackage  Timber
- * @since    Timber 0.1
+ * @link https://developer.wordpress.org/themes/basics/template-hierarchy/
  */
 
-$context         = Timber::context();
-$timber_post     = Timber::get_post();
-$context['post'] = $timber_post;
+namespace Timber\StarterTheme;
+
+use Timber\Timber;
+
+$post    = Timber::get_post();
+$context = Timber::context(
+	[
+		'post' => $post,
```

Comment:
We already have a `post` filled in context, when we use it on a singular template, right?